cmd/util.go

```diff
@@ -36,3 +36,18 @@ func ifFilesPassed(files []string) error {
 	}
 	return nil
 }
+
+// RemoveDuplicateFiles function will see input files and will remove duplicate entries
+func RemoveDuplicateFiles(inputFiles []string) []string {
```

Comment:
Does this have to be an exported function?

Reply:
no :(